### PR TITLE
Update MAINTAINERS.md

### PR DESCRIPTION
**Pull request by thisisnotapril**
Moved 21 maintainers to emeritus status; updated affiliation for @bogdandrutu

### CODE REVIEW

**Comment by thisisnotapril** [comment]
@markdroth @srini100 for your review

File: MAINTAINERS.md [hunk 1]
```diff
@@ -10,52 +10,31 @@ for general contribution guidelines.
 ## Maintainers (in alphabetical order)
 - [a11r](https://github.com/a11r), Google LLC
 - [apolcyn](https://github.com/apolcyn), Google LLC
-- [arjunroy](https://github.com/arjunroy), Google LLC
-- [AspirinSJL](https://github.com/AspirinSJL), Google LLC
-- [bogdandrutu](https://github.com/bogdandrutu), Google LLC
+- [bogdandrutu](https://github.com/bogdandrutu), Splunk
```
Comment (markdroth) [inline MAINTAINERS.md:13]: I don't believe that Bogdan still works on gRPC.  I think he move to emeritus.

File: MAINTAINERS.md [hunk 2]
```diff
@@ -10,52 +10,31 @@ for general contribution guidelines.
...
-- [jboeuf](https://github.com/jboeuf), Google LLC
 - [jiangtaoli2016](https://github.com/jiangtaoli2016), Google LLC
-- [jkolhe](https://github.com/jkolhe), Google LLC
 - [jtattermusch](https://github.com/jtattermusch), Google LLC
-- [karthikravis](https://github.com/karthikravis), Google LLC
-- [kumaralokgithub](https://github.com/kumaralokgithub), Google LLC
 - [lidizheng](https://github.com/lidizheng), Google LLC
 - [markdroth](https://github.com/markdroth), Google LLC
 - [matthewstevenson88](https://github.com/matthewstevenson88), Google LLC
 - [mehrdada](https://github.com/mehrdada), Dropbox, Inc.
```
Comment (markdroth) [inline MAINTAINERS.md:25]: I think Mehrdad can move to emeritus as well.

File: MAINTAINERS.md [hunk 3]
```diff
@@ -10,52 +10,31 @@ for general contribution guidelines.
...
 - [sanjaypujare](https://github.com/sanjaypujare), Google LLC
-- [sheenaqotj](https://github.com/sheenaqotj), Google LLC
 - [soheilhy](https://github.com/soheilhy), Google LLC
 - [sreecha](https://github.com/sreecha), LinkedIn
 - [srini100](https://github.com/srini100), Google LLC
 - [stanley-cheung](https://github.com/stanley-cheung), Google LLC
 - [veblush](https://github.com/veblush), Google LLC
-- [vishalpowar](https://github.com/vishalpowar), Google LLC
-- [Vizerai](https://github.com/Vizerai), Google LLC
 - [vjpai](https://github.com/vjpai), Google LLC
```
Comment (markdroth) [inline MAINTAINERS.md:36]: Vijay can probably move to emeritus now as well.

File: MAINTAINERS.md [hunk 4]
```diff
@@ -10,52 +10,31 @@ for general contribution guidelines.
...
-- [nathanielmanistaatgoogle](https://github.com/nathanielmanistaatgoogle), Google LLC
 - [nicolasnoble](https://github.com/nicolasnoble), Google LLC
 - [pfreixes](https://github.com/pfreixes), Skyscanner Ltd
-- [qixuanl1](https://github.com/qixuanl1), Google LLC
 - [ran-su](https://github.com/ran-su), Google LLC
-- [rmstar](https://github.com/rmstar), Google LLC
 - [sanjaypujare](https://github.com/sanjaypujare), Google LLC
-- [sheenaqotj](https://github.com/sheenaqotj), Google LLC
 - [soheilhy](https://github.com/soheilhy), Google LLC
 - [sreecha](https://github.com/sreecha), LinkedIn
```
Comment (markdroth) [inline MAINTAINERS.md:32]: Sree can move to emeritus.

**Review by markdroth — APPROVED** [review]
A few more folks that should probably move to emeritus.  Otherwise, looks good.

**Pull request closed without merging**